.github/workflows/python-package.yml

```diff
@@ -28,5 +28,5 @@ jobs:
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=79 --statistics
+        # no --exit-zero
```

Comment:
`# no --exit-zero` - i don't think we need this comment, I'll remove it

Reply:
Ok, after change this, this PR may be merged.